examples/DisposableTokens/Program.cs

```diff
@@ -0,0 +1,42 @@
+﻿using System;
+using Momento.Sdk;
+using Momento.Sdk.Auth;
+using Momento.Sdk.Auth.AccessControl;
+using Momento.Sdk.Config;
+using Momento.Sdk.Responses;
+
+ICredentialProvider authProvider = new EnvMomentoTokenProvider("MOMENTO_AUTH_TOKEN");
+
+IAuthClient client = new AuthClient(AuthConfigurations.Default.Latest(), authProvider);
+var scope = new DisposableTokenScope(Permissions: new List<DisposableTokenPermission>
+{
+    new DisposableToken.CacheItemPermission(
+        CacheRole.ReadWrite,
+        CacheSelector.ByName("cache"),
+        CacheItemSelector.AllCacheItems
+    ),
+    new DisposableToken.CachePermission(
+        CacheRole.ReadOnly,
+        CacheSelector.ByName("topsecret")
+    ),
+    new DisposableToken.TopicPermission(
+        TopicRole.PublishSubscribe,
+        CacheSelector.ByName("cache"),
+        TopicSelector.ByName("example-topic")
+    )
+});
```

Comment:
in a subsequent PR it would be good to add another example (can just be later in this same file) that uses the shorthand sugar APIs in `DisposableTokenScopes`.